sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -184,35 +178,38 @@ DocumentAnalysisClient documentAnalysisClient = new DocumentAnalysisClientBuilde
 ## Key concepts
 ### DocumentAnalysisClient
 The [DocumentAnalysisClient][document_analysis_sync_client] and [DocumentAnalysisAsyncClient][document_analysis_async_client]
-provide both synchronous and asynchronous operations
- - Recognizing document fields and layout using custom models built to analyze your custom documents.
- These values are returned in a `AnalyzeResult` object. See example [Analyze Custom Documents](#analyze-documents-using-a-custom-model).
- - Recognizing document layout, including tables, lines and words, without the need to build a model.
- Document layout is returned in a `AnalyzeResult` object. See example [Analyze Layout](#analyze-layout).
-- Recognizing common fields from the following document types using prebuilt models. These fields and meta-data are returned
-  in a collection of fields on a `AnalyzeResult` object.
-  Supported prebuilt models:
-    - Receipts
-    - Business cards
-    - Invoices
-    - Identity Documents
-    - Documents
-    
-  See example [Prebuilt Models](#use-prebuilt-models).
+provide both synchronous and asynchronous operations for analyzing input documents using custom and prebuilt models
+through the `beginAnalyzeDocument` and `beginAnalyzeDocumentFromUrl` methods.
+Use the `model` parameter to select the type of model for analysis.
+
+|Model| Features
+|-|-
+|"prebuilt-layout"| Text extraction, selection marks, tables
+|"prebuilt-document"| Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices"| Text extraction, selection marks, tables, and prebuilt fields and values pertaining to English invoices
+|"prebuilt-businessCard"| Text extraction and prebuilt fields and values pertaining to English business cards
+|"prebuilt-idDocument"| Text extraction and prebuilt fields and values pertaining to US driver licenses and international passports
+|"prebuilt-receipt"| Text extraction and prebuilt fields and values pertaining to English sales receipts
+|"{custom-model-id}"| Text extraction, selection marks, tables, labeled fields and values from your custom documents
+
+Sample code snippets to illustrate using a DocumentAnalysisClient [here](#analyze-documents-using-a-custom-model "Analyze Documents Using a Custom Model").
```

Comment:
what about pointing to the [Samples Readme](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/src/samples) instead of a section here that doesn't show all the samples for the DocumentAnalysisClient?